docs/projects/README.md

```diff
@@ -0,0 +1,453 @@
+stakeholders to the project. They provide visibility into our current
+initiatives and up to date information on each initiative. Each project thread
+is a place to see the current status, timeline, and motivation of a given work
+stream.
+
+All requirements for working with project threads can be found in this document.
+If you have any questions please feel free to reach out to an Openverse
+maintainer in the #openverse channel of the
+[Making WordPress Chat](https://make.wordpress.org/chat) or
+[file an issue](https://github.com/WordPress/openverse/issues/new/choose).
+
+## Where documents are kept, how they are named, and how they are reviewed
+
+For each project, planning documents should be kept in the `rfcs/` directory of
+this repository. Each project should have its own subdirectory. Projects
+predating this document have been updated to follow this structure. Individual
+documents should be date-stamped in the `YYYYMMDD` format. An example of this
+entire structure follows:
+
+```
+| rfcs/
+   | service_metrics/
+      - YYYYMMDD-project_proposal.md
+      - YYYYMMDD-implementation_plan.md
+   | 3d_model_support/
+      - YYYYMMDD-project_proposal.md
+      - YYYYMMDD-implementation_plan_(catalogue).md
+      - YYYYMMDD-implementation_plan_(api).md
+      - YYYYMMDD-implementation_plan_(frontend).md
+```
+
+Using subdirectories makes it slightly easier to navigate an ever-growing list
+of planning documents. This is especially the case in projects similar to the 3D
+Models project in the example above that may require multiple implementation
+planning documents for each part of the application. In the process of planning
+or implementing some projects, we may develop additional supporting documents or
+appendixes that go beyond the required scope of the project planning process
+described in this document. Those documents should accompany the required ones
+in the same subdirectory for the project.
+
+All project documentation can be authored with any tool, but must be written in
+[GitHub Flavored Markdown](https://github.github.com/gfm/) including the
+available
+[GitHub-specific advanced formatting features](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting).
+
+Additionally, all documents should be formatted to pass the Prettier linting
+step required by the repository for Markdown documents.
+
+When submitting a new document for a project, the author should open a pull
+request with the document. The review process of the document will happen as a
+PR review. Once the document is accepted and approved, it is merged and the
+relevant step of the process is complete.
+
+## Providing project updates
+
+At all stages of a project, weekly public updates are the best way for Openverse
+contributors and maintainers to understand the health of a project. Importantl,
+these updates allow us to allocate resources to projects that need them before
```

Comment:
I appreciate the desire to avoid dehumanizing corporate-speak, but I think "resources" might be appropriate here. Resources as in "human resources" typically refers to _people_ themselves, not actions people might take. We're referring to the later here: code review, development, pair programming, but also non-people things like a software licenses, books or educational materials. If you're able to think of alternative wording that bypasses this problem, I'm certainly open to changing this, but the specific phrase "allocating people"would be more objectifying, to me.

Reply:
I don't think "allocating people" is good either, FWIW. I just think resources is quite vague and usually read this as you said, inhumane corporate speak ("human resources"). All the things you mentioned are either time or money based, so maybe "allocate more time or material resources" would be appropriate. It is clearer and specific about what is available without grouping people or people's energy into the concept of "resources".

Just a nitpick, I guess. If no one else has an issue with it, then feel free to ignore.

Reply:
Updated to use the "allocate more time or material resources" language.